cmd/influxd/run.go

```diff
@@ -23,126 +25,213 @@ import (
 	"github.com/influxdb/influxdb/udp"
 )
 
-func Run(config *Config, join, version string) (*messaging.Broker, *influxdb.Server, *raft.Log) {
-	log.Printf("influxdb started, version %s, commit %s", version, commit)
+type RunCommand struct {
+	// The logger passed to the ticker during execution.
+	Logger    *log.Logger
+	logWriter *os.File
+	config    *Config
+	hostname  string
+	server    *Server
+}
 
-	var initBroker, initServer bool
-	if initBroker = !fileExists(config.BrokerDir()); initBroker {
-		log.Printf("Broker directory missing. Need to create a broker.")
+func NewRunCommand() *RunCommand {
+	return &RunCommand{
+		server: &Server{},
 	}
+}
+
+type Server struct {
```

Comment:
Server is an overload term in our system. "server" sometimes refers to just a data node. Would just "node" be a better name? Thoughts?